### PR TITLE
fix: dont show bridge route

### PR DESCRIPTION
**Pull request by estebanmino**
Fixes BX-1547
Figma link (if any):

not showing routes on bridge flow as ticket describes

## What changed (plus any additional context for devs)

image

### CODE REVIEW

File: src/entries/popup/pages/swap/SwapSettings/SwapSettings.tsx [hunk 1]
```diff
@@ -324,66 +326,70 @@ export const SwapSettings = ({
...
-                        <Lens
-                          onKeyDown={() =>
-                            simulateClick(routesTriggerRef?.current)
-                          }
-                          style={{ borderRadius: 6, padding: 4 }}
+                    {!bridge ? (
+                      <Box
+                        testId="swap-settings-route-row"
+                        style={{ height: '32px' }}
+                      >
```
Comment (magiziz) [inline src/entries/popup/pages/swap/SwapSettings/SwapSettings.tsx:333]: Looks good. I usually prefer doing like this:

```tsx
{bridge && <Box>...</Box>}
```

File: lavamoat/build-webpack/policy.json [hunk 2]
```diff
@@ -1156,8 +1156,13 @@
...
-        "jest>@jest/core>jest-snapshot>@babel/traverse>@babel/generator>@jridgewell/trace-mapping>@jridgewell/sourcemap-codec": true,
-        "webpack>terser-webpack-plugin>@jridgewell/trace-mapping>@jridgewell/resolve-uri": true
+        "jest>@jest/core>jest-snapshot>@babel/traverse>@babel/generator>@jridgewell/trace-mapping>@jridgewell/resolve-uri": true,
+        "jest>@jest/core>jest-snapshot>@babel/traverse>@babel/generator>@jridgewell/trace-mapping>@jridgewell/sourcemap-codec": true
+      }
+    },
+    "jest>@jest/core>jest-snapshot>@babel/traverse>@babel/generator>@jridgewell/trace-mapping>@jridgewell/resolve-uri": {
+      "globals": {
+        "define": true
       }
```
Comment (magiziz) [inline lavamoat/build-webpack/policy.json:1166]: Needed ?

**Review by magiziz — APPROVED** [review]

File: src/entries/popup/pages/swap/SwapSettings/SwapSettings.tsx [hunk 3]
```diff
@@ -446,8 +452,12 @@ export const SwapSettings = ({
...
                 </Button>
               </Box>
-              <Box style={{ width: '102px' }}>
-                <Separator color="separatorTertiary" strokeWeight="1px" />
+              <Box alignItems="center" justifyContent="center" display="flex">
+                <Separator
+                  color="separatorTertiary"
+                  strokeWeight="1px"
+                  width="102px"
+                />
```
Comment (DanielSinclair) [inline src/entries/popup/pages/swap/SwapSettings/SwapSettings.tsx:460]: Added this little separator alignment change I noticed.

**Review by DanielSinclair — APPROVED** [review]
lgtm

**Pull request merged**